lib/trino-cache/src/main/java/io/trino/cache/EvictableCache.java

```diff
@@ -248,7 +269,6 @@ int tokensCount()
     @Override
     public void invalidate(Object key)
     {
-        invalidations.incrementAndGet();
```

Comment:
why?

Reply:
the invalidations counter-based approach to the load-after-timed-expire didn't work correctly and was replaced with a different approach